interfearence:
    hardware_interface:
        joints:
          - left_wheel_joint
          - right_wheel_joint
gazebo_ros_control:
    pid_gains:
        left_wheel_joint:
            p: 0
            i: 0
            d: 0
        right_wheel_joint:
            p: 0
            i: 0
            d: 0
